src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -467,6 +424,22 @@ void reshape_to_static_encoder(std::shared_ptr<ov::Model> model, const size_t fe
     model->reshape(new_shapes);
 }
 
+void reshape_input_ids(std::shared_ptr<ov::Model> model, const uint32_t input_size) {
+    std::map<std::string, ov::PartialShape> new_shapes;
+    for (auto input : model->inputs()) {
+        const auto& input_name = input.get_any_name();
+        ov::PartialShape new_shape;
+        if (input_name.find("input_ids") != std::string::npos) {
+            new_shape = ov::PartialShape({1, input_size});
+        } else {
+            new_shape = input.get_partial_shape();
```

Comment:
Can you just pass one layer to `reshape` function? It's weird you need to duplicate layers that are not supposed to be changed

Reply:
fixed